### PR TITLE
Break out trending/feed

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description
Break out trending & feed into their respective page containers. This is an ancient legacy thing we've had for a long time because feed & trending were originally rendered in the same page.

This also removes some react spring animations we no longer need because of it.

Changes against stage on ray.audius.co

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?
Touches feed & trending

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

Viewed feed & trending on both mobile & desktop.
Tested playback, feed filters, trending genre filters, trending time ranges, etc.

### CODE REVIEW

File: src/containers/feed-page/store/actions.js [hunk 1]
```diff
@@ -3,10 +3,6 @@ export const FETCH_SUGGESTED_FOLLOW_USERS =
 export const FOLLOW_USERS = 'DISCOVER/FOLLOW_USERS'
 export const SET_SUGGESTED_FOLLOWS = 'DISCOVER/SET_SUGGESTED_FOLLOWS'
 export const SET_FEED_FILTER = 'DISCOVER/SET_FEED_FILTER'
```
Comment (jowlee) [inline src/containers/feed-page/store/actions.js:5]: Rename `DISCOVER` to `FEED`

**Review by jowlee — APPROVED** [review]
played around w/ it and everything seems fine!

File: src/containers/App.js [hunk 2]
```diff
@@ -573,7 +574,7 @@ class App extends Component {
                 exact
                 path={TRENDING_PAGE}
                 render={() => (
-                  <Discover
+                  <TrendingPage
                     feedIsMain={false}
```
[the remark this answers is not in the document]
Comment (raymondjacobson) [reply src/containers/App.js:578]: ty!

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/r-aud-556

**Pull request merged**